docs/reference/commandline/attach.md

```diff
@@ -48,11 +48,11 @@ a container and leave it running using the `CTRL-p CTRL-q` key sequence.
 > so.
 
 It is forbidden to redirect the standard input of a `docker attach` command
-while attaching to a tty-enabled container (i.e.: launched with `-t`).
+while attaching to a TTY-enabled container (i.e., launched with `-i` and `-t`).
```

Comment:
WDYT about losing the i.e. for readability, I think it is implied in a aside like: "(using the `-i` or `-t` options)" or even "(`-i`, `-t` options).